pom.xml

```diff
@@ -787,52 +803,35 @@
                                 <!-- We enable nearly all doclint checks,
                                 except that we don't care about missing Javadoc
                                 on non-public classes and members. -->
-                                <arg>-Xdoclint:all</arg>
-                                <arg>-Xdoclint:missing/protected</arg>
+                                <arg>-Xdoclint:all,-missing/package</arg>
```

Comment:
This change is not just "cleanup": the previous two-argument variant no longer works on JDK 11.